scrc/dataset_creation/dataset_creator.py

```diff
@@ -137,17 +146,18 @@ class DatasetCreator(AbstractPreprocessor):
     Also creates the necessary files for a kaggle dataset and a huggingface dataset.
     """
 
-    def __init__(self, config: dict):
+    def __init__(self, config: dict, debug: bool = True):
         __metaclass__ = abc.ABCMeta
         super().__init__(config)
         self.logger = get_logger(__name__)
 
+        self.debug = debug
         self.seed = 42
         self.minFeatureColLength = 100  # characters
         self.debug_chunksize = 100
         self.real_chunksize = 1_000_000
         self.counter = 0
-        self.start_years = {"train": 2002, "validation": 2016, "test": 2018, "secret_test": 2020}
+        self.start_years = {Split.TRAIN.value: 2002, "validation": 2016, Split.TEST.value: 2018, Split.SECRET_TEST.value: 2020}
```

Comment:
please also use the split enum for validation

Reply:
Although it's "val" and not "validation" ?